lib/my/my_list.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  # rubocop:disable Style/ExplicitBlockArgument
+  def each
+    @list.each { |item| yield item }
+  end
```

Comment:
- [ ] Your `each` method does not work quite as expected. Please kindly fix so that it yields all members of the list in succession.